salt/metalk8s/addons/prometheus-operator/deployed/chart.sls

```diff
@@ -53429,7 +53429,7 @@ spec:
         - --path.sysfs=/host/sys
         - --path.rootfs=/host/root
         - --web.listen-address=$(HOST_IP):9100
-        - --collector.diskstats.ignored-devices=^(ram|fd|(h|s|v)d[a-z]|nvme\\d+n\\d+p)\\d+$
+        - --collector.diskstats.ignored-devices=^(ram|fd)\\d+$
```

Comment:
Ignore this line as this comes from the Prometheus stack version bump (I forgot to re-render after having changed the values in `charts/kube-prometheus-stack.yaml`)